### PR TITLE
Student child validation

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Change student, aged between 16 and 19 and not a child validation to be a soft validation instead:
image

### CODE REVIEW

File: app/models/form/sales/pages/student_not_child_value_check.rb
Comment (natdeanlewissoftwire) [inline app/models/form/sales/pages/student_not_child_value_check.rb:1]: same as my other comment about starting with `person_` but it feels even more tentative here

File: app/models/form/sales/questions/student_not_child_value_check.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        {
+          "student_not_child_value_check" => 0,
+        },
+        {
+          "student_not_child_value_check" => 1,
+        },
+      ],
+    }
+    @check_answers_card_number = person_index
+    @header = "Are you sure this person is not child?"
```
Comment (natdeanlewissoftwire) [inline app/models/form/sales/questions/student_not_child_value_check.rb]: grammar seems odd - maybe "Are you sure this person is not a child?"

File: app/services/imports/sales_logs_import_service.rb [hunk 2]
```diff
@@ -242,7 +243,8 @@ def fields_not_present_in_softwire_data
          staircase_bought_value_check
          monthly_charges_value_check
          hodate_check
-         saledate_check]
+         saledate_check
+         student_not_child_value_check]
```
Comment (natdeanlewissoftwire) [inline app/services/imports/sales_logs_import_service.rb:247]: nit: can we put the ] on the next line for tidiness

**Review by natdeanlewissoftwire — COMMENTED** [review]
small grammar point, otherwise looks good!

**Review by natdeanlewissoftwire — APPROVED** [review]

File: app/services/imports/sales_logs_import_service.rb (hunk 2 above)
Comment (kosiakkatrina) [reply app/services/imports/sales_logs_import_service.rb:247]: Oh, rubocop don't like that :<

**Pull request merged**